README.md

```diff
@@ -83,7 +83,7 @@ Set `isSecure` (default `true`) to `false` if you are using standard http. Take
 ```javascript
 var options = {
     cookieOptions: {
-        isSecure: process.env.NODE_ENV === 'development' ? false : true,
+        isSecure: process.env.NODE_ENV !== 'development',
```

Comment:
When was this change made? It's absolutely awful. I'd like to see this changed back to `true`. Defaulting to no security based on environment flags?! That's nuts.